.github/workflows/reusable-python-publish_to_artifactory.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish condan package to Artifactory 
+
+on:
+  workflow_call:
+    inputs:
+        package_name:
+            description: 'Name of the package to build'
+            required: true
+            type: string
+        python_ver:
+            description: 'Python version to use (eg: 3.10)'
+            required: true
+            type: string
+        version_tag:
+            description: 'Version tag of the package to build'
+            required: true
+            type: string
+    secrets:
+        EXTRA_REPO_TOKEN:
```

Comment:
**suggestion:** rename secret

we always need a token to publish to conda, so not really "extra"
Maybe `MIRA_CONDA_REPO_TOKEN`

Or should it be a parameter, so that each individual repo could publish to different repos that might need different tokens?

Reply:
I was thinking we could add an other variable like `EXTRA_REPO_NAME`. With it, we could reuse this workflow to publish in any jfrog artifactory conda repo (the official one and the dev one, for example). 

I'm okay to rename the secrets `MIRA_CONDA_REPO_TOKEN`.

Reply:
do you mean to publish to several repos at once? yes, that would be ideal
Maybe an array as input then? and the calling workflows would pass the relevant secrets in the array?

Reply:
I was thinking of applying this workflow several times, but you could also use an array. I think a json (or yaml) would be more appropriate. We could pass it as a secret, putting in the repository name (or url) and the secret. This would be a good way of working on different repositories simultaneously and having different tokens (like on the official repository and the development repository, for example).

Reply:
the benefit of passing the array instead of running the workflow is that it will push the exact same build to all the repos.
It will all save some execution time